Makefile

```diff
@@ -167,8 +167,8 @@ gen-test-golden: test-unit
 # dependencies to the go.mod file. To avoid that we change to a directory
 # without a go.mod file when downloading the following dependencies
 
-$(GOX):
-	(cd /; go install github.com/mitchellh/gox@v1.0.2-0.20220701044238-9f712387e2d2)
+$(GORELEASER):
+	(cd /; GOTOOLCHAIN=go1.24.6 go install github.com/goreleaser/goreleaser@latest)
```

Comment:
Could we inherit from existing defined Go version to avoid having to change it ?